listenbrainz_spark/request_consumer/request_consumer.py

```diff
@@ -80,7 +80,12 @@ def push_to_result_queue(self, messages):
                     self.rabbitmq.close()
                     self.connect_to_rabbitmq()
                     self.init_rabbitmq_channels()
-        current_app.logger.debug("Done!")
+
+        avg_size_of_message //= num_of_messages
```

Comment:
Sorry, this is a little confusing, where is avg_size initially defined?

Reply:
Sorry for that, incorrect `git add -p`. Fixed it.